source/includes/_acts-api-reference.md

```diff
@@ -47,9 +47,9 @@ During Auth0 authentication, the client passes 'client id' and 'client secret' t
 
 All API requests must be made over HTTPS, and API requests without authentication will fail. Security on data is managed by the ACTS platform, and API requests will provide the same admin access available to logged in ACTS admin users. 
 
-The user authentication endpoint uses a POST (with input parameters described above) to generate the access token [bearer token]. 
+The user authentication endpoint uses a POST to generate the access token [bearer token]. 
 
->Authentication API Endpoint - https: // dev-intelex.us.auth0.com/oauth/token 
+>Authentication API Endpoint - https: // {auth0 Sub-Domain}.auth0.com/oauth/token 
```

Comment:
I this we probably just need to mention that you receive instruction on how to authenticate from Intelex Consultant initially, and from the ACTS Web in the future, rather than specifying it here. Also I'd say any details about auth0 should be hidden i.e we should not mention Auth0 in our docs. For the URLs, I think we need to add custom intelex domain to Auth0.

Reply:
@heydarli made changes to the URL based on the comments from Anton . You can take a look and provide your approval .